tests/system-containers/main.yml

```diff
@@ -417,8 +437,7 @@
 
     - name: Install etcd
       command: >
-        atomic install
-        --system
+        {{ atomic_install_cmd }}
```

Comment:
Need double quotes here.

Reply:
I think the `>` notation for multi-line strings handles the variable expansion auto-magically without quotes.

Reply:
Good to know!